pkgs/build-support/fetchrepoproject/default.nix

```diff
@@ -40,7 +63,6 @@ stdenv.mkDerivation {
   ];
 
   buildInputs = [git gitRepo cacert] ++ optional (gnupg != null) [gnupg] ;
```

Comment:
Should not this have been `optional (gnupg != null) gnupg` or `optionals (gnupg != null) [gnupg]`?